File: Question4/gravityFlip.py
def gravityFlip(direction, arr):
    if (direction == 'L'):
        dir = 1
    if (direction == 'R'):
        dir = 0
    arr.sort(reverse=dir)
    return arr

if __name__ == '__main__':
    print(gravityFlip('L', [3, 2, 1, 2]))

File: Question4/GravityFlip.js
function Gflip(d,box){
    if(d === "L"){
        return box.sort(function(a, b){return b-a});
    }
    return box.sort(function(a, b){return a-b});

 
}
console.log(Gflip("R", [3, 2, 1, 2]))
console.log(Gflip('L', [1, 1, 5, 4, 3]))

